client/src/state/campaign/tests/CampaignActions-test.js

```diff
@@ -1,59 +0,0 @@
-/* global jest, jasmine, describe, it, expect, beforeEach */
-
-import configureMockStore from 'redux-mock-store';
```

Comment:
This test was removed because 'redux-mock-store' wasn't compatible with latest version of redux. There was a link in the failed test to a page where the maintainers or redux recommend testing with react-testing-library instead.  Since that kind of acts like a quasi behat test, and the fact we already have behat tests for campaign admin, I think we can safety just delete this test without a replacement